projects/solid/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@maskito/solid",
+    "version": "0.0.1",
+    "description": "",
+    "license": "MIT",
+    "author": "",
+    "type": "module",
+    "module": "dist/index.js",
+    "scripts": {
+        "start": "jiti ./src/index.ts",
+        "build": "tsup"
+    },
+    "devDependencies": {
+        "jiti": "^1.21.0",
+        "solid-js": "^1.8.22",
+        "tsup": "^8.0.2",
+        "typescript": "^5.3.3"
```

Comment:
TypeScript is already mentioned in the root `package.json`:
https://github.com/taiga-family/maskito/blob/52824cd49487e2cab9c46038294777c015c65efa/package.json#L126